src/modules/27-platform/auth-settings/pages/Configuration/EnablePublicAccess/__tests__/EnablePublicAccess.test.tsx

```diff
@@ -27,17 +27,21 @@ describe('EnablePublicAccess', () => {
     })
     ;(setPublicAccessPromise as jest.Mock).mockImplementationOnce(mockSetPublicAccessPromise)
 
-    const { getByTestId, queryByText } = render(
+    const { getByTestId } = render(
       <TestWrapper>
         <EnablePublicAccess enabled={false} refetchAuthSettings={refetchAuthSettings} canEdit={true} />
       </TestWrapper>
     )
 
     const switchBtn = getByTestId('toggle-enable-public-access')
     await userEvent.click(switchBtn!)
-    await waitFor(() => expect(setPublicAccessPromise).toBeCalled())
 
-    expect(queryByText('platform.authSettings.updatedPublicAccess')).toBeTruthy()
+    const dialogContainer = findDialogContainer()
+    expect(dialogContainer).toBeTruthy()
```

Comment:
Instead of using `toBeTruthy`, think about it from a user's point of view. `toBeInTheDocument` or `toBeVisible` do the same thing, but convey what you're trying to prove.
```suggestion
    expect(dialogContainer).toBeInTheDocument()
```

Reply:
Done